test/integration/swap/common.js

```diff
@@ -0,0 +1,81 @@
+import { expect } from 'chai'
+import MetaMaskConnector from 'node-metamask'
+import { Client, providers, crypto } from '../../../'
```

Comment:
Use files from `src` (instead of `dist`) so it can be reported back to coveralls.

Reply:
changed.